.github/actions/spelling/expect.txt

```diff
@@ -0,0 +1,493 @@
+Abf
+Abh
+acbf
+addressparser
+AHyperlink
+alekstorm
+algo
+allcaps
+amd
+amlog
+anonymize
+Aopen
+api
+applypatch
+apps
+AQws
+args
+argv
+ARIo
+Arnfj
+asciidoc
+attrs
+autocrlf
+autolearn
+automerge
+autosquash
+avarab
+aws
+backend
+basedon
+BAYES
+bbox
+BCAE
+bcbb
+bfbde
+BFk
+bigfilethreshold
+bigint
+Bjarmason
+BKBc
+BKBd
+BKL
+blockquote
+blocksource
+BNew
+bordercolor
+borderopacity
+brk
+BTr
+buf
+bugfix
+busybox
+BXx
+CABPp
+cacheinfo
+callbackfn
+callchain
+CBGFIg
+ccd
+CDcd
+CEu
+charset
+chdir
+CIimap
+CIPh
+CIsmtp
+CIsmtphost
+CIsmtpopts
+CIsmtppass
+cleanbranch
+cmds
+cmp
+cmt
+coccinelle
+codeaurora
+codepath
+colabti
+commonjs
+compat
+concat
+config
+Cpjh
+creativecommons
+cscscscc
+Ctw
+cummings
+CURLOPT
+cwd
+cygwin
+CYKD
+daf
+dasharray
+dbd
+dcmitype
+dco
+dddf
+deduplicate
+destructuring
+devnull
+devops
+DFEEDBEEF
+diffcore
+diffopt
+diffs
+difftool
+dirname
+dirs
+DKIM
+dns
+docname
+dotdot
+dotgitx
+downlevel
+dscho
+Dtn
+dugite
+Duy
+eab
+ECDHE
+eef
+EHLO
+Emacs
+emails
+EOH
+eol
+eqeqeq
+EQlr
+EQON
+esac
+eslint
+eslintrc
+ESMTP
+ESMTPSA
+ESNEXT
+EWJ
+facebook
+fallthrough
+Falsy
+fba
+fbee
+fcc
+feedbeef
+feedeadbeeffeeddeadbeef
+filesystem
+filetypes
+FJNt
+fooba
+Fqo
+freenode
+FRX
+fsck
+fsf
+fsmonitor
+fstat
+fullref
+funcname
+funcs
+FVYA
+fwlink
+Fxww
+GAgy
+genno
+gettext
+ggg
+gitattributes
+gitconfig
+gitgit
+gitgitgadget
```

Comment:
I would normally move this into `allow.txt` (a supplemental dictionary), but hadn't reached that stage yet.